### DISCUSSION TITLE
Does not work with Expo 50

### DISCUSSION THREAD

**Discussion by Sommerzeit** [iddan/react-native-canvas#328] [Announcements]
Hi there!

I had no problems using `react-native-canvas` with Expo 49. After upgrading to Expo 50, the whole app is quiting on me as soon as I interact with the canvas in any way.

Is there any way to fix this? I would really like to use the newest Expo version.

Kind regards!

**Comment by aaron-hutchinson-dev** [comment]
@Sommerzeit 
Solution:
https://stackoverflow.com/a/41956442

I upgraded to Expo 50 and was getting this canvas error:
[ReferenceError: Property 'regeneratorRuntime' doesn't exist]

The top answer linked above solved the issue for me.

In summary:
1. Run `npm install --save babel-polyfill`

2. Add the following to your index.js (or other entry point):
  `import 'babel-polyfill';`
